cmd/fyne/internal/commands/package.go

```diff
@@ -421,3 +427,15 @@ func validateAppID(appID, os, name string, release bool) (string, error) {
 
 	return appID, nil
 }
+
+func hasGoCode(dir string) bool {
+	found := false
+	_ = filepath.Walk(dir+string(filepath.Separator), func(path string, fi fs.FileInfo, err error) error {
+		if fi.IsDir() || filepath.Ext(path) != ".go" {
+			return err
+		}
+		found = true
```

Comment:
This solution means that we are looping through the whole source tree even if we found a single .go file. I think there is a specific error you can return to terminate early? Looks like filepath.SkipAll maybe?

Reply:
Got it working on 1.19 with a custom error that's ignored anyways. Thanks for making me look again :-).